modules/apps/portal-template/portal-template-freemarker/src/main/java/com/liferay/portal/template/freemarker/internal/RestrictedLiferayObjectWrapper.java

```diff
@@ -218,15 +217,12 @@ else if (object instanceof BaseModel) {
 			if (currentCompanyId != CompanyConstants.SYSTEM) {
 				BaseModel<?> baseModel = (BaseModel<?>)object;
 
-				Map<String, Function<Object, Object>> getterFunctions =
-					(Map<String, Function<Object, Object>>)
-						(Map<String, ?>)baseModel.getAttributeGetterFunctions();
+				Map<String, Object> modelAttributes =
+					baseModel.getModelAttributes();
```

Comment:
@ericyanLr @nikolettabuza The reason we did not use "baseModel.getModelAttributes();" is it is too heavy, it need to calculate the values for all attributes, and then only use one of hem (companyId). I will take a look at this issue next week.

Tina.